blackbox/docs/sql/general/value-expressions.rst

```diff
@@ -169,6 +169,34 @@ optionally prefixed with ``ARRAY``::
 
     :ref:`data-type-array-literals`
 
+.. _sql_expressions_array_subquery:
+
+Another way to construct an array is by using an ``ARRAY(subquery)`` expression
+as part of the :ref:`Select list <sql_reference_select_list>` of a ``SELECT`` statement::
+
+    ARRAY '(' subquery ')'
+
+Example::
+
+   cr> select mountain, height,
```

Comment:
For an example it might make sense to stick to something a bit simpler. How about only `select array(select height from sys.summits order by height limit 5)`

Reply:
good point - done